testing/environments/cloud/docker_image.auto.tfvars.sample

```diff
@@ -0,0 +1,2 @@
+docker_image_override={"elasticsearch":"docker.elastic.co/cloud-release/elasticsearch-cloud-ess","kibana":"docker.elastic.co/cloud-release/kibana-cloud","apm":"docker.elastic.co/observability-ci/elastic-agent"}
+docker_image_tag_override={"elasticsearch":"8.6.0-f96862e0-SNAPSHOT","kibana":"8.6.0-f96862e0-SNAPSHOT","apm":"mysampletag"}
```

Comment:
```suggestion
docker_image_override={"elasticsearch":"docker.elastic.co/cloud-release/elasticsearch-cloud-ess","kibana":"docker.elastic.co/cloud-release/kibana-cloud","agent":"docker.elastic.co/observability-ci/elastic-agent"}
docker_image_tag_override={"elasticsearch":"8.6.0-f96862e0-SNAPSHOT","kibana":"8.6.0-f96862e0-SNAPSHOT","agent":"mysampletag"}
```

Is using "apm" to control the agent version actually necessary? Can we rename it to "agent' in the Terraform files? This would make it much more obvious to people who aren't familiar with integrations server or know that these files were copied from APM.